README.md

```diff
@@ -1,37 +1,70 @@
 # trey_visay
+
 Trey Visay (ត្រីវិស័យ) is a mobile application that allows users whom study in high school from grade 10 to 12 to understand about their strength, weakness, behavior, favorites subject and so on. Moreover, This app allows users to get online consultant in what major they should study and which schools or universities have provides to that major. Students can view majors’ videos in this app too.
 
-# Environment
-Trey Visay has built using React Native. And below are its environment.
+## Features
 
-Pre-requisites:
-Xcode version 9 or above
+- Career assessment
+- Personality assessment
+- Institution information
+- Vocational jobs information
+- Video jobs information
+- Sign up, sign in, and profile
+- Share app
+- About application
 
-react: 16.6.0-alpha.8af6728
+## Install
 
-react-native: 0.57.4
+Just clone the repo:
 
-Node: 8.11.3
+```
+git clone git@github.com:ilabsea/trey-visay.git
+```
 
-Yarn: 1.12.3
+Install all dependencies:
 
-npm: 5.6.0
+```
+npm install
+```
+Link all dependencies:
 
-Watchman: 4.5.0
+```
+react-native link
+```
 
-Cocoapods v1.6 or above
+For ios:
 
-You can find out more Trey Visay’s dependencies in trey-visay/package.json.
+```
+cd ios && pod install
+```
+That's it. Its ready to go.
+
+## Run
+
+Run on android:
 
-# Running on iOS
-Install all React Native package dependencies 
-$ npm install
+```
+react-native run-android
+```
 
-Load React Native link for the project:
-$ react-native link
+Run on ios:
+
+```
+react-native run-ios
+```
+## Environment
```

Comment:
there is iOS pre-requisite where no Android, we have to add info on Android development too.